crates/bevy_pbr/src/render/mesh_bindings.rs

```diff
@@ -49,11 +58,27 @@ mod layout_entry {
             render_device,
         )
     }
-    pub(super) fn skinning(binding: u32) -> BindGroupLayoutEntry {
-        buffer(binding, JOINT_BUFFER_SIZE as u64, ShaderStages::VERTEX)
+    pub(super) fn skinning(render_device: &RenderDevice, binding: u32) -> BindGroupLayoutEntry {
+        let is_storage = render_device.limits().max_storage_buffers_per_shader_stage > 0;
```

Comment:
```suggestion
        let is_storage = render_device.limits().max_storage_buffers_per_shader_stage > 0;
```

At this point, I think we should add a method like "storage_buffers_supported()" to RenderDevice.

Reply:
What does your suggestion change? The lines look identical to me.

Reply:
I didn't change anything sorry. This was meant to be a regular comment on that line.